stock-filters/ChirpVillage/PathGeneration.py

```diff
@@ -66,22 +66,38 @@ def find_paths_BFS(self, start, end):
 
         # Draw paths
         current = self.add(end, path[end])
+        direction = path[end]
         while current != start:
             # find next in path
             x, z = current
             block = self.surface.surface_map[x][z]
             block.type = Block.PATH
+            # # If path his moving horiziontally, try to widen it by adding blocks above and below it
+            # if (direction == (-1,0) or direction == (1,0)):
```

Comment:
Commented out code

Reply:
@johnnyb8 can this be removed? Its from the liveshare session

Reply:
this was deliberate, basically, there are two ways of doing this 
- the current way which just fills in all its neighboring blocks
- check which direction the path is going and fill in either the top/bottom or left/right neighbors

Reply:
So do we want to keep this commented out code in the codebase? In case we want to change the method were using, or for reference? Not sure if we want to submit it with commented out code?

Reply:
Keeping for now. @johnnyb8 can remove later before submission once we get a good working submission.